docs/blog/_posts/2020-04-28-24th-dotty-milestone-release.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.24.0-RC1 - 2.13.2 standard library, better error messages and more
+author: Anatolii Kmetiuk
+authorImg: /images/anatolii.png
+date: 2020-04-28
+---
+
+Hello! We are excited to announce 0.24.0-RC1 of Dotty. In this version, we have updated the standard library to 2.13.2. Also, we have made some work to make error messages more user friendly and a bunch of other polishings to the language.
+
+You can try out this version right now, from the comfort of your SBT, by visiting the [home page](https://dotty.epfl.ch/) and scrolling down to the "Create a Dotty Project" section.
+
+Alternatively, you can try this version of Scala online via [Scastie](https://scastie.scala-lang.org/). Once you're there, click "Build Settings" and set "Target" to "Dotty".
+
+Enjoy the ride🚀!
+
+<!--more-->
+# REPL works with indented code
+REPL now supports indented code. Consider the following snippet:
+
+```scala
+scala> if true then
+     |   print(1)
+     |   print(2)
+     |
+```
+
+Previously, the REPL would have stopped after `println(1)`. Now, it waits either for an `else` block or an extra newline to indicate the end of the expression. The above example will output `12` as expected.
+
+# Better error message for ifs that miss an else branch
+The error messages are now more beginner-friendly. Consider the following:
+
+```scala
+def f: Int = if ??? then 1
+```
+
+Above, the `if` expression returns a `Unit` since an `else` clause is missing. Previously, the user would have gotten the following error:
+
+```
+-- [E007] Type Mismatch Error: ...
+12 |def f: Int = if ??? then 1
+   |             ^^^^^^^^^^^^^
+   |             Found:    Unit
+   |             Required: Int
+```
+
+Now, the above error message also contains the following sentence:
+
+```
+   |             Maybe you are missing an else part for the conditional?
+```
+
+We hope this change will make the language more intuitive for new users.
+
+# Inline overrides
```

Comment:
@odersky @nicolasstucki could you verify if everything is alright with this piece?

Reply:
Maybe we should merge https://github.com/lampepfl/dotty/pull/8785 before the release